### PR TITLE
fix(lint): critical bug with pylint usage

### PR DESCRIPTION
**Pull request by MrJs133**
The previous pylint configuration had issues, which prevented pylint from being triggered properly. 

This PR has modified the pylint configuration to fix the lingering issues and fixed residual pylint issues (ML LLM)

Details are as follows:
- fixed incubator-hugegraph-ai/style/code_format_and_analysis.sh
- fixed incubator-hugegraph-ai/hugegraph-ml/src/hugegraph_ml
- fixed incubator-hugegraph-ai/hugegraph-llm/src/hugegraph_llm

### CODE REVIEW

File: hugegraph-ml/src/hugegraph_ml/models/bgrl.py [hunk 1]
```diff
@@ -252,4 +252,5 @@ def get_graph_drop_transform(drop_edge_p, feat_mask_p):
     if feat_mask_p > 0.0:
         transforms.append(FeatMask(feat_mask_p, node_feat_names=["feat"]))
 
-    return Compose(transforms)
\ No newline at end of file
+    return Compose(transforms)
+
```
Comment (imbajin) [inline hugegraph-ml/src/hugegraph_ml/models/bgrl.py:256]: lack empty line here?

File: hugegraph-ml/src/hugegraph_ml/models/cluster_gcn.py [hunk 2]
```diff
@@ -36,9 +35,9 @@ class SAGE(nn.Module):
     def __init__(self, in_feats, n_hidden, n_classes):
         super().__init__()
         self.layers = nn.ModuleList()
-        self.layers.append(dglnn.SAGEConv(in_feats, n_hidden, "mean"))
-        self.layers.append(dglnn.SAGEConv(n_hidden, n_hidden, "mean"))
-        self.layers.append(dglnn.SAGEConv(n_hidden, n_classes, "mean"))
+        self.layers.append(dglnn.SAGEConv(in_feats, n_hidden, "mean")) # pylint: disable=E1101
+        self.layers.append(dglnn.SAGEConv(n_hidden, n_hidden, "mean")) # pylint: disable=E1101
+        self.layers.append(dglnn.SAGEConv(n_hidden, n_classes, "mean")) # pylint: disable=E1101
```
Comment (imbajin) [inline hugegraph-ml/src/hugegraph_ml/models/cluster_gcn.py:40]: maybe change it to the `method` scope?

File: style/code_format_and_analysis.sh [hunk 3]
```diff
@@ -53,7 +53,5 @@ if [ "$PYLINT" = true ] ; then
   echo "[pylint] Start code analysis and check,
   we need to manually fix all the warnings mentioned below before commit! "
   export PYTHONPATH=${ROOT_DIR}/hugegraph-llm/src:${ROOT_DIR}/hugegraph-python-client/src:${ROOT_DIR}/hugegraph-ml/src
-  pylint --rcfile=${ROOT_DIR}/style/pylint.conf ${ROOT_DIR}/hugegraph-llm
-  pylint --rcfile=${ROOT_DIR}/style/pylint.conf ${ROOT_DIR}/hugegraph-ml
-  pylint --rcfile=${ROOT_DIR}/style/pylint.conf --disable C0103 ${ROOT_DIR}/hugegraph-python-client
+  pylint --rcfile=${ROOT_DIR}/style/pylint.conf ${ROOT_DIR}/hugegraph-llm ${ROOT_DIR}/hugegraph-ml ${ROOT_DIR}/hugegraph-python-client
```
Comment (imbajin) [inline style/code_format_and_analysis.sh:56]: @simon824 root reason for "CI not work as expect" before 😢

**Review by imbajin — COMMENTED** [review]
Overall good with tiny improve

File: hugegraph-ml/src/hugegraph_ml/models/bgrl.py (hunk 1 above)
Comment (MrJs133) [reply hugegraph-ml/src/hugegraph_ml/models/bgrl.py:256]: emmm
yes, you are right

File: hugegraph-ml/src/hugegraph_ml/models/cluster_gcn.py (hunk 2 above)
Comment (MrJs133) [reply hugegraph-ml/src/hugegraph_ml/models/cluster_gcn.py:40]: OK!

File: hugegraph-ml/src/hugegraph_ml/models/bgrl.py (hunk 1 above)
Comment (MrJs133) [reply hugegraph-ml/src/hugegraph_ml/models/bgrl.py:256]: yes, you are right!

**Review by imbajin — APPROVED** [review]

**Pull request merged**